packages/server/graphql/mutations/createReflection.ts

```diff
@@ -75,6 +75,7 @@ export default {
       content: normalizedContent,
       plaintextContent,
       entities,
+      groupColor: '#FFFFFF',
```

Comment:
i'm curious, why is the color a part of the reflection?

Reply:
Before I added `groupColor` here and on `Reflection` I was getting this `TypeError` when I ran the `typecheck` script: 
```
parabol-server: database/types/Reflection.ts(42,7): error TS2339: Property 'groupColor' does not exist on type 'ReflectionInput'.
parabol-server: database/types/Reflection.ts(42,7): error TS6133: 'groupColor' is declared but its value is never read.
parabol-server: graphql/types/RetroReflection.ts(56,18): error TS2339: Property 'groupColor' does not exist on type 'Reflection'.
```
I believe it's because as reflections are added, they're expecting a `groupColor` to assign a value now. The reason I have it being initialized to `'#FFFFFF'` is so there won't be a flash of another color before it get's set to the reflection's assigned `groupcolor`.